data.sql

```diff
@@ -4,3 +4,10 @@ INSERT INTO animals VALUES (1, 'Agumon', '2020-03-02',0,true,10.23)
 INSERT INTO animals VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8 )
 INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04)
 INSERT INTO animals VALUES (4, ' Devimon', '2017-05-17', 5, true, 11)
+INSERT INTO animals VALUES (5, ' Charmander', '2020-02-08', 0, false, -11)
+INSERT INTO animals VALUES(6, 'Plantmon', '2021-11-15', 2, true, -5.7),
+(7, 'Squirtle', '1993-04-02', 3, false,  -12.13),
+(8, 'Angemon', '2005-06-12', 1, true, -45),
+(9, ' Boarmon', '2007-06-07', 7, true, 20.4),
```

Comment:
You have made a typo with the date_of_birth of Boarmon. It is 2005/06/07 and not 2007/06/07. Kindly fix this mistake.